README.markdown

```diff
@@ -31,6 +33,13 @@ The original soundtrack and PDF edition are also available in
 Chapter 8: Heaven's Harp was never recovered. I suspect \_why started drawing it,
 but never finished or published.
 
+> "Quite so," he said.  "I went out to my car yesterday and there was a shoe in the passenger's
```

Comment:
Could you add `* * *` above the quote so that it renders a horizontal rule that will visually separate it from the “Notes” section? Thanks